examples/reference/widgets/FileInput.ipynb

```diff
@@ -127,6 +134,91 @@
+    "3. BokehJS (or the libraries that it uses) may fail at creating the message from a very (very!) large string\n",
+    "4. `Tornado` imposes two limits on the data being transferred: (1) on the maximum size of a buffer and (2) on the maximum size of the websocket message. (1) is 100 MB by default (set by Tornado) and (2) is 20 MB by default (set by Bokeh).\n",
+    "\n",
+    "#### Increase the limits\n",
+    "\n",
+    "While there's nothing much that can be done about 1., 2. and 3. (except informing your users), the limits defined by Tornado and Bokeh can be overwritten.\n",
+    "\n",
+    "##### Server context\n",
+    "\n",
+    "In a server context your application must be executed with `python your_app.py` (because `panel serve` doesn't allow to configure all the options provided by Bokeh and Tornado):\n",
+    "\n",
+    "```python\n",
+    "# your_app.py\n",
+    "import panel as pn\n",
+    "\n",
+    "app = ...\n",
+    "\n",
+    "MAX_SIZE_MB = 150\n",
+    "\n",
+    "pn.serve(\n",
+    "    app,\n",
+    "    # Increase the maximum websocket message size allowed by Bokeh\n",
+    "    websocket_max_message_size=MAX_SIZE_MB*1024*1014,\n",
+    "    # Increase the maximum buffer size allowed by Tornado\n",
+    "    http_server_kwargs={'max_buffer_size': MAX_SIZE_MB*1024*1014}\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "##### Notebook context\n",
+    "\n",
+    "In a Jupyter notebook (classic or lab) the limits of Tornado (Tornado's Web Sockets are already used by the Jupyter notebook for communication purposes) can be set in a configuration file. The default maximum buffer size is 512 MB and the default maximum websocked message size is 10 MB.\n",
+    "\n",
+    "*Classic Notebook:*\n",
+    "\n",
+    "Generate a configuration file with `jupyter notebook --generate-config` and update it with:\n",
+    "\n",
+    "```python\n",
+    "c.NotebookApp.tornado_settings = {\"websocket_max_message_size\": 150 * 1024 * 1024}\n",
+    "c.NotebookApp.max_buffer_size = 150 * 1024 * 1024\n",
+    "```\n",
+    "\n",
+    "*Lab:*\n",
+    "\n",
+    "Generate a configuration file with `jupyter lab --generate-config` and update it with:\n",
+    "\n",
+    "```python\n",
+    "c.ServerApp.tornado_settings = {'websocket_max_message_size': 150 * 1024 * 1024}\n",
+    "c.ServerApp.max_buffer_size = 150 * 1024 * 1024\n",
+    "```\n",
+    "\n",
+    "#### Caveats\n",
+    "\n",
+    "* The maximum sizes set in either Bokeh or Tornado refer to the maximum size of the **message** that is transferred through the web socket connection, which is going to be larger than the actul size of the uploaded file since the file content is encoded in a `base64` string. So if you set a maximum size of 100 MB for your application, you should indicate your users that the upload limit is a value that is **less** than 100 MB.\n",
```

Comment:
```suggestion
    "* The maximum sizes set in either Bokeh or Tornado refer to the maximum size of the **message** that is transferred through the web socket connection, which is going to be larger than the actual size of the uploaded file since the file content is encoded in a `base64` string. So if you set a maximum size of 100 MB for your application, you should indicate your users that the upload limit is a value that is **less** than 100 MB.\n",
```